README.md

```diff
@@ -69,6 +69,10 @@ Options:
 - `maxHeaderSize`, the maximum length of request headers in bytes. 
   Default: `16384` (16KiB)
 
+- `headersTimeout`, the amount of time the parser will wait to receive the complete
+  HTTP headers (Node 12 and above only).
```

Comment:
Node 14

Reply:
I think it might be because the use the new parser, so the different behavior would still be there when the old-parser option is used?

Reply:
The approach to headersTimeout measurement was changed in Node 13.13 (PR: https://github.com/nodejs/node/pull/32329). I think this is the root cause of the difference between Node 12 and 14.